packages/bootstrap/scss/_variables.scss

```diff
@@ -2984,6 +2984,21 @@ $gantt-task-selected-text: $selected-text !default;
 $gantt-task-selected-border: $selected-bg !default;
 $gantt-task-progress-selected-bg: $selected-bg !default;
 
+$gantt-delayed-bg: $error !default;
+$gantt-delayed-bg-lighter: tint($gantt-delayed-bg, 5) !default;
+$gantt-advanced-bg: $success !default;
+$gantt-advanced-bg-lighter: tint($gantt-advanced-bg, 5) !default;
+
+$gantt-planned-color: $accent !default;
+$gantt-planned-dependencies-offset: .6em !default;
+$gantt-planned-task-line-height: .75em !default;
+$gantt-planned-margin-bottom: .35em !default;
+$gantt-planned-border-width: 2px !default;
+$gantt-planned-moment-dimension: 6px !default;
```

Comment:
suggestion: use $gantt-planned-moment-width and $gantt-planned-moment-height variables